src/scripts/quests/QuestLine.ts

```diff
@@ -61,7 +65,11 @@ class QuestLine {
         this.quests.push(quest);
     }
 
-    beginQuest(index = 0, initial?: number) {
+    beginQuest(index = 0, initial?: number, force = false) {
+        if (index == 0 && this.optional && !force) {
+            this.state(QuestLineState.onBulletinBoard);
+            return;
+        }
```

Comment:
I'm a little confused as to what this does that it wasn't doing before?

Reply:
Sets the state to onBulletinBoard when started. Does that answer your question?

Reply:
I feel like we should just be using requirements for that, rather than updating it's state?